tests/TestWireless.py

```diff
@@ -12,3 +13,26 @@ def setUp(self):
     def test_import(self):
         # if this module loads, then the import worked...
         self.assertTrue(hasattr(Wireless, 'connect'))
+
+
+class TestCMD(unittest.TestCase):
+    """
+        Tests against cmd function.
+    """
+    def setUp(self):
+        self.com = cmd('echo "test_ok"')
+        self.empty_com = cmd('echo -n')
+
+    def test_cmdcomparission(self):
+        """
+            Check if we can test against the output of
+            the current test command
+        """
+        self.assertTrue('test_ok' in self.com)
```

Comment:
Why not test for the full, exact output? Is there other text coming along in `self.com`?